src/components/TracePage/TraceTimelineViewer/VirtualizedTraceView.js

```diff
@@ -386,4 +446,4 @@ function mapDispatchToProps(dispatch) {
   return bindActionCreators(actions, dispatch);
 }
 
-export default connect(mapStateToProps, mapDispatchToProps)(VirtualizedTraceView);
+export default connect(mapStateToProps, mapDispatchToProps)(VirtualizedTraceViewImpl);
```

Comment:
@saminzadeh You had a suggestion for a better name than `*Impl` (where Impl is short form of implementation)... What was that, again?